controllers/spec/common.go

```diff
@@ -412,9 +412,7 @@ func getLegacyDownloadCommand(downloadPath, componentPackage string, tlsProvided
 		if tlsProvided {
 			args = append(args, []string{
 				"--tls-allow-insecure",
-				"${tlsAllowInsecureConnection:-" + DefaultForAllowInsecure + "}",
 				"--tls-enable-hostname-verification",
-				"${tlsHostnameVerificationEnable:-" + DefaultForEnableHostNameVerification + "}",
```

Comment:
since `PulsarAdmin` doesn't support parameter in format: `--tls-allow-insecure true` but only support `--tls-allow-insecure` 

and the shell command will be very long if we judge the value of `tlsAllowInsecureConnection`, so we use hardcoded `--tls-allow-insecur` and `--tls-enable-hostname-verification` here